blocks/api/factory.js

```diff
@@ -10,20 +10,24 @@ import { get, castArray, findIndex, isObjectLike } from 'lodash';
 import { getBlockSettings } from './registration';
 
 /**
- * Returns a block object given its type and attributes
+ * Returns a block object given its type and attributes.
  *
  * @param  {String} blockType   BlockType
  * @param  {Object} attributes  Block attributes
  * @return {Object}             Block object
  */
 export function createBlock( blockType, attributes = {} ) {
+	// Get the type definition associated with a registered block.
 	const blockSettings = getBlockSettings( blockType );
 
+	// Do we need this? What purpose does it have?
```

Comment:
Let's create a separate issue for this. I think our current use-cases don't really need it as much as it seems they would.